### PR TITLE
LPS-145789 Create new tests for specificFields method

### PR DESCRIPTION
**Pull request by YurenaC**
Internal task to add test to the specificFields method that is used to generate the excel file with the content dashboard items
[Ticket](https://issues.liferay.com/browse/LPS-145789)

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by YurenaC** [comment]
ci:test:sf

File: modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/action/test/GetContentDashboardItemInfoMVCResourceCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Before
+	public void setUp() throws Exception {
+		_group = GroupTestUtil.addGroup(
```
Comment (ealonso) [inline modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/action/test/GetContentDashboardItemInfoMVCResourceCommandTest.java:71]: @YurenaC I think you can use a simply method like this:

`_group = GroupTestUtil.addGroup();`

File: modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/action/test/GetContentDashboardItemInfoMVCResourceCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+			TestPropsValues.getCompanyId(), TestPropsValues.getUserId(), 0,
+			"Test Site");
+
+		_serviceContext = ServiceContextTestUtil.getServiceContext(
+			_group.getGroupId());
+	}
+
+	@Test
+	public void testGetFileSpecificFields() throws Exception {
+		FileEntry fileEntry = DLAppLocalServiceUtil.addFileEntry(
```
Comment (ealonso) [inline modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/action/test/GetContentDashboardItemInfoMVCResourceCommandTest.java:81]: @YurenaC here you can use DLTestUtil

**Comment by YurenaC** [comment]
ci:test:sf

**Comment by YurenaC** [comment]
ci:test:sf

**Comment by YurenaC** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**